internal/util/cmdutil/cmdutil.go

```diff
@@ -88,14 +89,17 @@ func ResolveAbsAndRelPaths(path string) (string, string, error) {
 // DockerCmdAvailable runs `docker ps` to check that the docker command is
 // available, and returns an error with installation instructions if it is not
 func DockerCmdAvailable() error {
+	op := errors.Op("docker.check")
+	suggestedText := `Docker is required to run this command.
+To install docker, follow the instructions at https://docs.docker.com/get-docker/
+`
+	buffer := &bytes.Buffer{}
+
 	cmd := exec.Command("docker", "version")
-	var stderr bytes.Buffer
-	cmd.Stderr = &stderr
+	cmd.Stderr = buffer
 	err := cmd.Run()
 	if err != nil {
-		return fmt.Errorf(stderr.String() +
-			`Docker is required to run this command. 
-To install docker, follow the instructions at https://docs.docker.com/get-docker/`)
+		return errors.E(op, suggestedText, fmt.Errorf("%s", suggestedText))
```

Comment:
This looks a bit strange. The way `errors.E` is currently implemented, we will drop either the error or the string.

Reply:
I was playing with `UserText` that I took it out and this is a residue from that iteration. Thanks for catching this. cleaned up.